tests/Unit/Helpers/ControlSystem/Examples.hpp

```diff
@@ -0,0 +1,137 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "ControlSystem/Protocols/ControlSystem.hpp"
+#include "ControlSystem/Protocols/Measurement.hpp"
+#include "ControlSystem/Protocols/Submeasurement.hpp"
+#include "ControlSystem/RunCallbacks.hpp"
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/LinkedMessageQueue.hpp"
+#include "Parallel/GlobalCache.hpp"
+#include "Parallel/Invoke.hpp"
+#include "ParallelAlgorithms/Actions/UpdateMessageQueue.hpp"
+#include "Time/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/ProtocolHelpers.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace control_system::TestHelpers {
+template <typename ControlSystem>
+struct ReplaceThisWithControlSystemComponentWhenThatIsWritten;
+
+struct SomeTagOnElement : db::SimpleTag {
+  using type = double;
+};
+
+struct MeasurementResultTag : db::SimpleTag {
+  using type = double;
+};
+
+struct SomeControlSystemUpdateFunctor {
```

Comment:
Change self-request: This is not a functor, and also does not need constexpr if guards.  Also check if any documentation refers to it as a functor.